cv32e20/bsp/Makefile

```diff
@@ -12,17 +12,17 @@ RISCV_GCC = $(RISCV_EXE_PREFIX)gcc
 RISCV_AR = $(RISCV_EXE_PREFIX)ar
 SRC = crt0.S handlers.S syscalls.c vectors.S
 OBJ = crt0.o handlers.o syscalls.o vectors.o
-LIBCV-VERIF = libcv-verif.a 
-CFLAGS ?= -Os -g -static -mabi=ilp32 -march=$(CV_SW_MARCH) -Wall -pedantic
+LIBCV-VERIF = libcv-verif.a
+CFLAGS ?= -Os -g -static -mabi=ilp32 -march=$(RISCV_MARCH) -Wall -pedantic
```

Comment:
For me it is fine to keep it as CV_SW but then it should be passed to the bsp makefile or use an `export` on the Makefile. As it is right now if the variable is not set in the environment it will be empty here.